app/script/audio/AudioRepository.js

```diff
@@ -149,15 +147,13 @@ window.z.audio.AudioRepository = class AudioRepository {
    * @returns {undefined}
    */
   _preload() {
-    Object
-      .keys(this.audio_elements)
-      .forEach((audio_id) => {
-        if (this.audio_elements.hasOwnProperty(audio_id)) {
-          const audio_element = this.audio_elements[audio_id];
-          audio_element.preload = 'auto';
-          audio_element.load();
-        }
-      });
+    Object.keys(this.audio_elements).forEach(audio_id => {
+      if (this.audio_elements.hasOwnProperty(audio_id)) {
+        const audio_element = this.audio_elements[audio_id];
+        audio_element.preload = 'auto';
+        audio_element.load();
+      }
+    });
```

Comment:
I like Prettier, but things like this are really a disadvantage. It makes the code not really more readable.

Reply:
I think it's quite minor compare to overall readability.. and these cases over time could be simplified anyway.